### PR TITLE
Update a-l package homepages

### PR DESCRIPTION
**Pull request by Zopolis4**
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/Zopolis4/chromebrew.git CREW_BRANCH=nowegot crew update
```

### CODE REVIEW

File: packages/gnome_icon_theme.rb [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 class Gnome_icon_theme < Autotools
   description 'GNOME Icon Theme'
-  homepage 'https://ftp.gnome.org/pub/GNOME/sources/gnome-icon-theme/'
+  homepage 'https://download.gnome.org/sources/gnome-icon-theme/'
```
Comment (uberhacker) [inline packages/gnome_icon_theme.rb:5]: This isn't really a home page.

File: packages/harfbuzz.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 class Harfbuzz < Meson
   description 'HarfBuzz is an OpenType text shaping engine.'
-  homepage 'https://www.freedesktop.org/wiki/Software/HarfBuzz/'
+  homepage 'https://github.com/harfbuzz/harfbuzz'
```
Comment (uberhacker) [inline packages/harfbuzz.rb:9]: Actually, this is the home page: https://harfbuzz.github.io/

File: packages/gnome_icon_theme.rb (hunk 1 above)
Comment (Zopolis4) [reply packages/gnome_icon_theme.rb:5]: I'm just following the redirect

File: packages/libgconf.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 class Libgconf < Autotools
   description 'GConf is a system for storing application preferences.'
-  homepage 'https://projects-old.gnome.org/gconf/'
+  homepage 'https://projects.gnome.org/gconf/'
```
Comment (uberhacker) [inline packages/libgconf.rb:5]: 404?

**Review by uberhacker — APPROVED** [review]

**Pull request merged**